packages/contracts/contracts/discovery/Project.sol

```diff
@@ -65,7 +68,11 @@ contract Project is IProject, ERC165 {
             IController(controller).hasProjectManagerRole(msg.sender),
             "not a project manager"
         );
+        _;
+    }
 
+    modifier onlyBatch() {
+        IController(controller).isProjectInBatch(address(this), msg.sender);
```

Comment:
this modifier isn't currently needed right now, is it? maybe it was a leftover as well?